src/TEKFrame.java

```diff
@@ -4,6 +4,8 @@
 import java.awt.Toolkit;
 import javax.swing.JScrollPane;
 import javax.swing.JPanel;
+import java.io.*;
+import java.awt.event.*;
```

Comment:
I've ran everything through and the list of imports would be as follows:

- import javax.swing.JFrame;
- import javax.swing.SwingUtilities;
- import java.awt.Dimension;
- import java.awt.Toolkit;
- import javax.swing.JScrollPane;
- import javax.swing.JToolBar;
- import java.util.ArrayList;
- import javax.swing.JMenuItem;
- import java.awt.event.ActionEvent;
- import java.awt.event.ActionListener;
- import javax.swing.JMenu;
- import javax.swing.JMenuBar;
- import java.io.IOException;
- import java.io.BufferedWriter;
- import javax.swing.JFileChooser;
- import java.io.BufferedReader;
- import java.awt.Point;
- import java.io.File;
- import java.io.FileReader;
- import javax.swing.JOptionPane;
- import java.io.FileWriter;